### DISCUSSION TITLE
[灌水] 发烧了好几天今天终于好了...

### DISCUSSION THREAD

**Discussion by BlakrPander** [share121/inter-knot#1220] [其他]
![IMG_3066](https://github.com/user-attachments/assets/1324c124-4047-441c-b2d4-7a48fb06baba)

放我自己养的薄荷镇楼（话说这玩意真是怎么养怎么有啊）
昨天烧退了才有力气爬起来去医院，检查了一下应该是炎症引起的发烧，医生说可能是上呼吸道感染了怎么的。反正在家里吃了好几天四季抗病毒胶囊就好了。不过这几天完全没学习...又凭空摆烂三四天，胡适之啊胡适之.jpg 
这几天我们这边风挺大，昼夜温差也挺大的，很容易感冒流感。换季期间绳网的大家也要注意安全啊！
总之身体是好了，先出去玩一上午再说嘿嘿

**Comment by 2Mnn** [comment]
抱抱绳匠，恭喜打败了病魔，出门在外都要注意身体哦孩子。你的小薄荷很可爱呢

**Comment by GanlinSama** [comment]
恭喜楼主

**Comment by Hotalou** [comment]
恭喜楼主！身体健康！
